code/nodes/modelGeneration/funcs.q

```diff
@@ -65,7 +65,7 @@ modelGeneration.modelPrep:{[cfg;mdlDict;tgt]
 // @return     {<} the appropriate function or projection in the case of sklearn
 modelGeneration.mdlFunc:{[lib;fnc;mdl]
   $[lib in key models;
-    get[".automl.models.",string[lib],".fitScore"];
+    util.qpyFuncSearch[".automl.models.",string[lib],".fitScore"];
```

Comment:
This I don't believe can be a python function as python functions can't be name spaced in `.automl` so get will suffice